### ISSUE TITLE
Can't set "cursor" CSS property on DraggableItem

### ISSUE THREAD

**Issue by kgoggin** [alexfedoseev/re-dnd#18]
I wanted to update the `cursor` for my Draggable Items to be `grab` and then `grabbing` while dragging, but it wasn't working. In doing some digging, I've found that adding `pointer-events: none` disables the use `cursor` (yay CSS 🙄).

I definitely understand the reasoning behind setting `pointer-events: none`, but I wonder if there'd be way to set it on the underlying child of DraggableItem instead of that `<div/>` itself, so you could still set the cursor prop of the div? 🤔

There may not be an easy/workable fix here, but I figured I'd file it anyway just to see if you had any ideas. Thanks!

**Comment by alex35mil** [comment]
Yeah, I remember I had fun times with exactly this issue. TBH I don't remember what I ended up with, most likely just stopped wasting time on this and moved on to something else. Let's keep it opened, I will look into this sometime.
